src/steamship/agents/service/agent_service.py

```diff
@@ -165,6 +202,9 @@ def run_agent(self, agent: Agent, context: AgentContext):
             logging.info(f"Emitting via function: {func.__name__}")
             func(action.output, context.metadata)
 
+        # Finally, mark this block as responded to.
```

Comment:
wdyt: should we record a log event related to marking that a response has been sent?